test/test_examples_2d_old.jl

```diff
@@ -189,15 +189,15 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
   end
   @testset "parameters_mhd_orszag_tang.toml with t_end=0.09" begin
     test_trixi_run(joinpath(EXAMPLES_DIR, "parameters_mhd_orszag_tang.toml"),
-            l2   = [0.2167754423280243, 0.26359006944514035, 0.31414664596581104, 0.0, 0.5121313736058165, 0.22945862419190388, 0.34334574741333446, 0.0, 0.006113771244085815],
-            linf = [1.2713060246638115, 0.6733000091062668, 0.8764507997673942, 0.0, 2.819437219022322, 0.6756993525030384, 0.9970376054894377, 0.0, 0.05310784504107611],
+            l2   = [0.21662313415818582, 0.2635698604231871, 0.31395699611730377, 0.0, 0.5122276249069517, 0.22914894367706035, 0.34302293430536107, 0.0, 0.0031837261356598232],
+            linf = [1.2455340346415893, 0.6656259804847943, 0.8530619473770993, 0.0, 2.762224683447692, 0.6641473992806939, 0.9631804383659317, 0.0, 0.04504842687596635],
             t_end = 0.09)
   end
   # second orszag-tang test added to exercise all components of flux_hll for GLM-MHD
   @testset "parameters_mhd_orszag_tang.toml with t_end=0.06" begin
```

Comment:
```suggestion
  @testset "parameters_mhd_orszag_tang.toml with flux_hll" begin
```
Here I'd rather highlight the different surface flux and not the end time